packages/sdk/src/types/DataConnectorTypes.ts

```diff
@@ -5,8 +5,8 @@ export type DataConnectorCapabilities = {
 };
 
 export type PageConfig = {
-    filters: DataFilter[];
-    sorting: DataSorting[];
+    filters?: DataFilter[] | null;
```

Comment:
should we update the type in studio-connectors package as well?

Reply:
we should indeed